src/backer/cardaning.py

```diff
@@ -120,6 +121,41 @@ def flushQueue(self):
             self.timer = Timer(90, self.flushQueue)
             self.timer.start()
 
+    def publishEventv2(self, event: bytearray):
+        self.pending_kel = event
```

Comment:
This used to be a list and is now a single event... just a note - I'm OK with it for now. But in a later PR we should work on the queuing

Reply:
added the TODO, will modify it at polling service ticker
ref here: 60b40541ac1ece5dff7ef05ea8725fb0c7342700